### PR TITLE
test: Update test/common for "medium" browser layout

### PR DESCRIPTION
**Pull request by mvollmer**
New pixels: https://github.com/cockpit-project/pixel-test-reference/compare/e45318da3c227623af5062326fab086d87f7e9cd..44585b3d10aebacd9d543c3efbb72e4d77102e32

### CODE REVIEW

**Comment by mvollmer** [comment]
I don't know what has changed in TestApplication-testRunImageSystem-integration-mobile-pixels.png...

**Comment by marusak** [comment]
> I don't know what has changed in TestApplication-testRunImageSystem-integration-mobile-pixels.png

Seems like colors has changed - that the button to submit is darker (or lighter, I never know which is new one :smile: ) Any idea how that happened?

File: Makefile [hunk 1]
```diff
@@ -168,7 +168,7 @@ bots:
 # when you start a new project, use the latest release, and update it from time to time
 test/common:
 	flock Makefile sh -ec '\
-	    git fetch --depth=1 https://github.com/cockpit-project/cockpit.git 748cfe36086cdd7069c698d0193a1b4751a1b80e; \
+	    git fetch --depth=1 https://github.com/cockpit-project/cockpit.git b49422a9c82858df0a0df74e7b8207472a2e9110; \
```
Comment (marusak) [inline Makefile:171]: We just released 262 - maybe we can use that already?

**Comment by mvollmer** [comment]
> > I don't know what has changed in TestApplication-testRunImageSystem-integration-mobile-pixels.png
> 
> Seems like colors has changed - that the button to submit is darker (or lighter, I never know which is new one smile ) Any idea how that happened?

I don't think that's it... it would have been red in the delta image then.  I try to figure it out...

File: Makefile (hunk 1 above)
Comment (mvollmer) [reply Makefile:171]: Good point!

**Comment by mvollmer** [comment]
> > > I don't know what has changed in TestApplication-testRunImageSystem-integration-mobile-pixels.png
> > 
> > 
> > Seems like colors has changed - that the button to submit is darker (or lighter, I never know which is new one smile ) Any idea how that happened?
> 
> I don't think that's it... it would have been red in the delta image then. I try to figure it out...

There is one pixel different in the info circle next to "Host port"...

File: Makefile (hunk 1 above)
Comment (mvollmer) [reply Makefile]: Done.

**Review by marusak — APPROVED** [review]
Thanks!

**Pull request merged**